### PR TITLE
Part5

### PR DESCRIPTION
**Pull request by cmurnick**

### CODE REVIEW

**Comment by cmurnick** [comment]
I added a dropdown option on the new/edit pages to add what relationship I have with the person.  It pushes up to firebase

File: index.html [hunk 1]
```diff
@@ -3,14 +3,15 @@
 <head>
 	<title>Angular Contacts</title>
 <link rel="stylesheet" type="text/css" href="lib/node_modules/bootstrap/dist/css/bootstrap.min.css">
+<link href="https://fonts.googleapis.com/css?family=Arapey" rel="stylesheet">
 <!--   <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"> -->
```
Comment (zoeames) [inline index.html]: no commented code.  use it or remove it

File: sass/_colors.scss [hunk 2]
```diff
@@ -1 +1,2 @@
-$blue: #42f4eb;
\ No newline at end of file
+$blue: #c2fcf6;
```
Comment (zoeames) [inline sass/_colors.scss:1]: be consistent with spacing.  make sure there are the same number of enters after each class.  no empty lines before closing curly brackets etc

**Comment by zoeames** [comment]
looks good 👍

**Pull request merged**